mcsema/BC/Function.cpp

```diff
@@ -260,6 +336,95 @@ static llvm::BasicBlock *GetOrCreateBlock(TranslationContext &ctx,
   return block;
 }
 
+
+static void CreateLandingPad(TranslationContext &ctx,
+                             struct NativeExceptionFrame *eh_entry) {
+  std::stringstream ss;
+  auto is_catch = (eh_entry->action_index != 0);
+  ss << "landingpad_" << std::hex << eh_entry->lp_ea;
+  auto landing_bb = llvm::BasicBlock::Create(
+      *gContext, ss.str(), ctx.lifted_func);
+
+  llvm::IRBuilder<> ir(landing_bb);
+  auto exn_type = llvm::StructType::get(llvm::Type::getInt8PtrTy(*gContext),
+                                        llvm::Type::getInt32Ty(*gContext),
+                                        nullptr);
+
+#if LLVM_VERSION_NUMBER > LLVM_VERSION(3, 6)
+  auto lpad = ir.CreateLandingPad(exn_type, 1, ss.str());
+#else
+  auto personality = gModule->getFunction(PERSONALITY_FUNCTION);
```

Comment:
This code is now out-of-date I think, as you'd want to do `auto personality_func = GetPer...`, and not use the `PERSONALITY_FUNCTION` macro.